### ISSUE TITLE
Can‘t work when server reply `posixrules` timezone, i.e.   Unknown time-zone ID: posixrules

### ISSUE THREAD

**Issue by baibaichen** [housepower/ClickHouse-Native-JDBC#333]
### Environment

- OS version: general
- JDK version:  openjdk 8
- ClickHouse Server version:  latest
- ClickHouse Native JDBC version: 2.5.3
- ClickHouse Server **OS** version:  suse 11.4
- (Optional) Spark version: N/A
- (Optional) Other components' version: N/A

### Error logs

![image](https://user-images.githubusercontent.com/3688732/112648731-c19c3800-8e84-11eb-9bac-0d2a2fc2dc3f.png)

This is because current jdk doesn't  support posixrules zoneid, but official JDBC supports it, see 
https://github.com/ClickHouse/clickhouse-jdbc/blob/master/clickhouse-jdbc/src/main/java/ru/yandex/clickhouse/ClickHouseConnectionImpl.java#L93 which uses Timezone

**Comment by pan3793** [comment]
TL;DR `posixrules` is deprecated, typically, we can treat it as `US/Eastern` for compatibility.
See detail at https://www.postgresql.org/docs/12/datetime-posix-timezone-specs.html
